ras_party/controllers/account_controller.py

```diff
@@ -410,7 +410,10 @@ def resend_verification_email(party_uuid, session):
     if not respondent:
         raise RasError(NO_RESPONDENT_FOR_PARTY_ID, status=404)
 
-    _send_email_verification(party_uuid, respondent.email_address)
+    if respondent.pending_email_address != '':
+        _send_email_verification(party_uuid, respondent.pending_email_address)
+    else:
+        _send_email_verification(party_uuid, respondent.email_address)
 
     return {'message': EMAIL_VERIFICATION_SENT}
```

Comment:
Is there any exception where the verification email would not be sent, if so worth putting this  if statement in a `try/except` block. Otherwise if the exception occurs, we are still gonna get the `EMAIL_VERIFICATION_SENT` message

Reply:
The only exception gets handled in _send_email_verification.  I've kept it in line with similar calls.